nextflow_schema.json

```diff
@@ -242,8 +242,8 @@
                 },
                 "save_reference": {
                     "type": "boolean",
-                    "description": "If generated by the pipeline save the BWA index in the results directory.",
-                    "help_text": "If the BWA index is generated by the pipeline use this parameter to save it to your results folder. These can then be used for future pipeline runs, reducing processing times.",
+                    "description": "If generated by the pipeline save the STAR index in the results directory.",
+                    "help_text": "If the STAR index is generated by the pipeline use this parameter to save it to your results folder. These can then be used for future pipeline runs, reducing processing times.",
```

Comment:
```suggestion
                    "help_text": "If an alignment index is generated by the pipeline use this parameter to save it to your results folder. These can then be used for future pipeline runs, reducing processing times.",
```

Reply:
We generate different indices for multiple tools so maybe good to keep it generic.